[IMP] purchase_discount: fix stock get_price_unit

cubells opened the pull request:
There's the concept of unit price in stock moves that it's used for stock valuation, and this one doesn't have the correct price when using discounts before this commit.

cc @Tecnativa

pedrobaeza commented on purchase_discount/models/purchase_order.py:
```diff
@@ -28,3 +28,24 @@ def _compute_amount(self):
+    _inherit = "purchase.order"
+
+    @api.model
+    def _prepare_inv_line(self, account_id, order_line):
```
This method is not needed on v9

pedrobaeza commented on purchase_discount/__openerp__.py:
```diff
@@ -9,6 +9,7 @@
               "ACSONE SA/NV,"
+              "AvanzOSC, S.L., "
               "Odoo Community Association (OCA)",
     "version": "9.0.1.0.1",
```
Change version number

pedrobaeza commented on purchase_discount/__openerp__.py:
```diff
@@ -9,6 +9,7 @@
               "Acysos S.L., "
               "Tecnativa, "
               "ACSONE SA/NV,"
+              "AvanzOSC, S.L., "
```
This little patch shouldn't be included in authors.

pedrobaeza committed:
[FIX] purchase_discount: Use correct price for stock calculations

There's the concept of unit price in stock moves that it's used
for stock valuation, and this one doesn't have the correct price when
using discounts before this commit.

sergio-teruel commented on purchase_discount/models/purchase_order.py:
```diff
@@ -28,3 +28,17 @@ def _compute_amount(self):
+        price = super(PurchaseOrderLine, self)._get_stock_move_price_unit()
+        if self.discount:
+            self.price_unit = price_unit
+        return price
```
Is multi?? Maybe you're missing a loop...

pedrobaeza replied:
No, it isn't. super method has self.ensure_one()

sergio-teruel replied:
Oks..
